packages/odie-client/src/components/message/get-support.tsx

```diff
@@ -52,8 +57,14 @@ export const GetSupport: React.FC< GetSupportProps > = ( {
 		chat,
 		isUserEligibleForPaidSupport: contextIsUserEligibleForPaidSupport,
 		canConnectToZendesk: contextCanConnectToZendesk,
+		trackEvent,
 	} = useOdieAssistantContext();
 
+	const mostRecentOpenConversation = useGetMostRecentOpenConversation();
+	const supportInteractionId = mostRecentOpenConversation?.metadata?.supportInteractionId || null;
```

Comment:
Would it be possible to return something like `userHasRecentOpenConversation` directly from the hook?